tests/containers/enable_mac.pm

```diff
@@ -0,0 +1,92 @@
+# SUSE's openQA tests
+#
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Package: containers
+# Summary: Enable SELinux or AppArmor
+#
+# Maintainer: QA-C team <qa-c@suse.de>
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use serial_terminal qw(select_serial_terminal);
+use Utils::Systemd qw(systemctl);
+use Utils::Logging 'save_and_upload_log';
+use bootloader_setup qw(replace_grub_cmdline_settings);
+use power_action_utils qw(power_action);
+use utils qw(zypper_call);
+use transactional qw(trup_call check_reboot_changes process_reboot);
+use version_utils qw(is_transactional);
+
+# Taken from tests/transactional/enable_selinux.pm
+sub check_enforcing {
+    assert_script_run('selinuxenabled');
+    validate_script_output("getenforce", sub { m/Enforcing/ });
+    validate_script_output("sestatus", sub { m/Current mode:.*enforcing/ });
+    validate_script_output("sestatus", sub { m/Mode from config file:.*enforcing/ });
+    record_info('SELinux', script_output('sestatus'));
+    record_info('Audit report', script_output('aureport'));
+    record_info('Audit denials', script_output('aureport -a', proceed_on_failure => 1));
+}
+
+sub run {
+    my ($self) = @_;
+
+    if (is_transactional) {
+        select_serial_terminal;
+    } else {
+        select_console 'root-console';
+    }
+
+    my $current_mac = script_output("grep -Eo '(selinux|apparmor)' /sys/kernel/security/lsm");
+    my $security_mac = get_required_var("SECURITY_MAC");
+    die "Invalid value for SECURITY_MAC: $security_mac" unless ($security_mac =~ /selinux|apparmor/);
+
+    return if ($security_mac eq $current_mac);
```

Comment:
I believe this requires a note for the test reviewer why the module has not done basically anything, as no switch is required. Or in case selinux is being already used, maybe it would be worth to check whether it is in enforcing mode, no?

Reply:
Not sure whether the default mode of AppArmor is enforcing thought

Reply:
The thorough check for SELinux is done later.  I will expand AppArmor though.

Reply:
Doesn't that check run only in case the SUT executed the switch? Because if the MACs are the same, the test will end over here and the SElinux can be set even in permissive mode.

Reply:
I do not know whether `/sys/kernel/security/lsm` reports `selinux` regardless of the selinux mode. So it seems that the test might exit prematurely.